File: 28-intro-to-js/04-classes.js
class SomeClass {

  constructor(name) {
    this.name = name
  }

  instanceMethod() {
    return `I am an intance method for ${this.name}`
  }

  static classVar = "I am a class var"

  static classMethod() {
    return `I am a class method`
  }

}

File: 28-intro-to-js/04-classes.rb
class SomeClass

  attr_accessor :name

  def initialize(name)
    self.name = name
  end

  def instance_method
    "I am an instance method for #{self.name}"
  end

  def self.class_method
    "I am a class method for #{self.name}"
  end

end
